src/sql/workbench/contrib/notebook/browser/notebookExplorer/notebookExplorerViewlet.ts

```diff
@@ -293,7 +306,7 @@ export class NotebookExplorerViewPaneContainer extends ViewPaneContainer {
 				let allViews = containerModel.allViewDescriptors;
 				allViews.forEach(v => {
 					let view = this.getView(v.id);
-					if (view !== this.searchView) {
+					if (view && view !== this.searchView) {
```

Comment:
Not sure why this would be necessary unless this.searchView could be undefined

Reply:
This was undefined for me when i was testing myself. Just removed this locally and got this:

 ERR Cannot read property 'setExpanded' of undefined: TypeError: Cannot read property 'setExpanded' of undefined